### PR TITLE
[202012][device/celestica]: Fix failed test cases of Haliburton platform API 

### PR DESCRIPTION
**Pull request by mudsut4ke**
**- Why I did it**
- To fix failed test cases of Haliburton platform APIs that found on [platform_tests](https://github.com/Azure/sonic-mgmt/tree/master/tests/platform_tests/api) script of 202012 version

**- How I did it**

- Picking a commit 4bf873bf4277912b11ff95c8100b5a40bb6cdbaa

> 1. Add device/celestica/x86_64-cel_e1031-r0/platform.json 
> 2. Update functions to support python3.7
> 3. Add more functions follow latest sonic_platform_base
> 4. Fix the bug
> 

 **- How to verify it**
- Run  [platform_tests](https://github.com/Azure/sonic-mgmt/tree/master/tests/platform_tests/api) script

**- Which release branch to backport (provide reason below if selected)**

- [x] 202012

Signed-off-by: Wirut Getbamrung [wgetbumr@celestica.com]

### CODE REVIEW

**Review by sujinmkang — APPROVED** [review]

**Comment by sujinmkang** [comment]
Looks good to me.

File: device/celestica/x86_64-cel_e1031-r0/sonic_platform/fan_drawer.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python
```
Comment (lguohan) [inline device/celestica/x86_64-cel_e1031-r0/sonic_platform/fan_drawer.py:1]: is this python3?

**Review by prgeor — APPROVED** [review]

**Comment by lguohan** [comment]
@mudsut4ke , can you get this pr to master branch and then we can backport to 202012 branch?

**Comment by mudsut4ke** [comment]
> @mudsut4ke , can you get this pr to master branch and then we can backport to 202012 branch?

@lguohan , I already has PR to master branch directly https://github.com/Azure/sonic-buildimage/pull/7579
But Ying said that the change on platform.json will break 202012 build due to platform.json format differences. https://github.com/Azure/sonic-buildimage/pull/7579#issuecomment-881616382

So I created this PR with change on platform.json

**Pull request merged**